monitor_workflow.rb

```diff
@@ -1,12 +1,46 @@
 require_relative './config.rb'
 
 require 'resque-scheduler'
+require 'faraday'
+require 'time'
+
+QUERY_RESULT_LIMIT = 10
+
 
 class MonitorWorkflow
   @queue = :job
 
   def self.perform
-    $log.info 'Monitoring workflow'
-    $log.info 'MP will implement me'
+    $log.info 'Checking for recent workflow results'
+
+    conn = Faraday.new(url: DAP_BASE_URL, ssl: {verify: false})
+    resp = conn.get(
+        '/api/v1/threat_assessment_runs',
+        {
+            private_token: PRIVATE_TOKEN,
+            sort_by: :start_date,
+            sort_order: :desc,
+            status: :finished,
```

Comment:
There are created_at timestamps in TA and TAR, however they are not returned by the REST API. I can live with the solution you implemented.